container/worker/conf/workers.ini

```diff
@@ -1,3 +1,3 @@
 [global]
 BACKEND = qemu
-HOST = http://openqa_webui
+HOST = http://nginx
```

Comment:
Why do you need to rename that? I would prefer a hostname of "openqa_webui" over nginx

Reply:
I didn't rename anything; I'm just making it connect to `nginx` rather than directly to the web UI. However, that only works when sharing the network with the web UI containers (see my last comment). So I'll have to change that somehow anyways. I suppose it would make sense to use the host IP/domain here as a default.

Reply:
I now changed it so the web UI containers and the worker container don't share the same network and are truly independent from each other. Being able to start the web UI containers + worker container on some machine and having them connect with each other is likely still a good default. So I put `172.17.0.1` here for to refer to the container host. This should be a good out-of-the-box experience. Of course I've been adding an explanatory comment what the IP address means and that one can put any host name there as usual.

By the way, putting `172.17.0.1` and an explaining comment there is definitely better than just `openqa_webui` because it is not at all clear whether `openqa_webui` is just an example or refers to an actual host name which is somehow made available by the setup¹.

¹ It isn't clear to me how it would work to connect via that host name so it was likely just an example.